pehlaschool/Classes/Games/ComprehensionTest/MultipleChoices/MultipleChoicesScene.cpp

```diff
@@ -120,7 +120,19 @@ namespace ComprehensionTest
                     break;
             }
             
-            string directionContent = LanguageManager::getInstance()->isEnglish() ? "âãè ©}æÚ ·UUUæ ¿ØÙ ·UUUUÚð¢Ð" : "Chagua jibu sahihi."; // Select the correct answer. // सही उत्तर का चयन करें।
+            string directionContent;
+            string langCode = LanguageManager::getInstance()->customLanguageCode;
+            if (langCode == "en") {
+                directionContent = "Select the correct answer.";
+            } else if (langCode == "hi") {
+                directionContent = "सही उत्तर का चयन करें।";
+            } else if (langCode == "ur") {
+                directionContent = "";
+            } else if (langCode == "bn") {
+                directionContent = "";
+            } else if (langCode == "sw") {
+                directionContent = "Chagua jibu sahihi.";
+            }
             
             if (_questionText.find("Choose:") != std::string::npos)
             {
```

Comment:
We need to add locale logic for this section.

Reply:
Added